db/config.cc

```diff
@@ -923,10 +926,12 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , wasm_udf_yield_fuel(this, "wasm_udf_yield_fuel", value_status::Used, 100000, "Wasmtime fuel a WASM UDF can consume before yielding")
     , wasm_udf_total_fuel(this, "wasm_udf_total_fuel", value_status::Used, 100000000, "Wasmtime fuel a WASM UDF can consume before termination")
     , wasm_udf_memory_limit(this, "wasm_udf_memory_limit", value_status::Used, 2*1024*1024, "How much memory each WASM UDF can allocate at most")
+    , relabel_file_name(this, "relabel_file_name", value_status::Used, "", "Optionally, read reable config from file")
```

Comment:
`relabel_config_file` would be a better name

Reply:
typo: `reable`

Reply:
will fix